### PR TITLE
German translation fix: typos, grammatical errors

### PR DESCRIPTION
**Pull request by MoritzLerch**
Some translations seemed to be from google translate, so I improved these. **Also used the typo-fixes from @bin101 provided in #218 (pull request).**

### CODE REVIEW

File: locales/active.de.toml [hunk 1]
```diff
@@ -312,19 +312,19 @@ other = "URL"
 
 ["message_handlers.handleNewGameMessage.embed.Title"]
 hash = "sha1-e48b15c3a0d35222115f14abddee1b3188c5755d"
-other = "Du hast gerade ein Spiel begonnen!"
+other = "Du hast gerade neues Spiel begonnen!"
```
Comment (bin101) [inline locales/active.de.toml:315]: ein neues Spiel ;)

File: locales/active.de.toml [hunk 2]
```diff
@@ -592,27 +592,27 @@ other = "Die Verzögerung beim Übergang von `{{.PhaseA}}` zu `{{.PhaseB}}` wurd
 
 ["settings.SettingDelays.wrongNumber"]
 hash = "sha1-6174db57aea459bfbe8f2b49296ca0a82c537f8f"
-other = "`{{.Number}}` ist keine gültige Zahl! Bitte versuche es erneut"
+other = "`{{.Number}}` ist keine gültige Zahl! Bitte versuch es erneut"
```
Comment (bin101) [inline locales/active.de.toml:595]: versuche ist schon richtig

File: locales/active.de.toml (hunk 1 above)
Comment (MoritzLerch) [reply locales/active.de.toml]: true ;-) Habs geändert.

File: locales/active.de.toml (hunk 2 above)
Comment (MoritzLerch) [reply locales/active.de.toml]: Mein Fehler ^^

**Review by bin101 — APPROVED** [review]
looks good to me now

**Pull request merged**